core/templates/dev/head/editor/ExplorationFeedback.js

```diff
@@ -161,23 +236,48 @@ oppia.controller('ExplorationFeedback', [
     {id: 'not_actionable', text: 'Not Actionable'}
   ];
 
-  $scope.getHumanReadableStatus = function(status) {
-    for (var i = 0; i < $scope.STATUS_CHOICES.length; i++) {
-      if ($scope.STATUS_CHOICES[i].id === status) {
-        return $scope.STATUS_CHOICES[i].text;
+  return {
+    STATUS_CHOICES: angular.copy(_STATUS_CHOICES),
+    getLabelClass: function(status) {
```

Comment:
Instead of doing something like `ng-class="getLabelClass(activeThread.status)"` couldn't you instead do: `ng-class="(activeThread.status === 'open') ? 'label label-info' : 'label label-default'"`

See http://stackoverflow.com/questions/18871277/adding-multiple-class-using-ng-class and http://stackoverflow.com/questions/7792652/what-is-the-best-way-to-conditionally-apply-a-class for more.